orttraining/orttraining/test/optimizer/shape_optimizer_test.cc

```diff
@@ -446,7 +446,7 @@ TEST(ShapeOptimizerTests, SliceGeneratesGraphOutput) {
     return Status::OK();
   };
 
-  std::vector<int> opset_candidates{10, 11, 12, 13, 14, 15};
+  std::vector<int> opset_candidates{10, 11, 12, 13, 14, 15, 20};
```

Comment:
@pengwa : Do we need to test for opsets 10, 11, 12, 13 and 14?

Reply:
to be clear I dont have a specific problem with testing for all the older opsets. The only advantage is it brings down CI time.